kernel/os/src/os_mbuf.c

```diff
@@ -370,6 +370,20 @@ _os_mbuf_copypkthdr(struct os_mbuf *new_buf, struct os_mbuf *old_buf)
     new_buf->om_data = new_buf->om_databuf + old_buf->om_pkthdr_len;
 }
 
+uint16_t
+os_mbuf_len(const struct os_mbuf *om)
+{
+    uint16_t len;
+
+    len = 0;
+    while (om != NULL) {
```

Comment:
Did you consider SLIST_FOREACH()? This is fine too.

Reply:
I don't think `SLIST_FOREACH` is applicable here since that macro requires an slist head.